suites/pre-release/dcf/4.preSignedURLTest.js

```diff
@@ -105,11 +120,11 @@ function performPreSignedURLTest(cloudProvider, typeOfTest, typeOfCreds) {
       const verificationMessage = typeOfTest === 'positive' ? `
                 a. The HTTP response code is Ok/200.
                 b. The response contain valid URLs to the files stored in AWS S3 or GCP Buckets.` : `
-                a. The HTTP response code is 401.
+                a. The HTTP response code is 404.
```

Comment:
I may be lost, but why did we switch to 404 instead of 401

Reply:
we switch to 404 as the guid is `NOT_FOUND`. The GUID isnt in the indexd, which causes the to get a 404 instead of 401